test/utils/InYearUtilSpec.scala

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package utils
+
+import org.scalamock.scalatest.MockFactory
+import support.UnitTest
+
+import java.time.LocalDateTime
+
+class InYearUtilSpec extends UnitTest
+  with MockFactory {
+
+  private val year2022: Int = 2022
+  private val month4: Int = 4
+  private val day5: Int = 5
+  private val day6: Int = 6
+  private val hour23: Int = 23
+  private val minute59: Int = 59
+
+  private def newDate(year: Int, month: Int, day: Int): LocalDateTime = LocalDateTime.of(year, month, day, 0, 0)
```

Comment:
It seems it is only used in line 45. Maybe we can just inline it?

Reply:
Done